src/client/cli/cmd/launch.cpp

```diff
@@ -47,15 +49,27 @@ const std::regex show{"s|show", std::regex::icase | std::regex::optimize};
 
 mp::ReturnCode cmd::Launch::run(mp::ArgParser* parser)
 {
-    auto ret = parse_args(parser);
-    if (ret != ParseCode::Ok)
+    petenv_name = Settings::instance().get(petenv_key);
+    if (auto ret = parse_args(parser); ret != ParseCode::Ok)
     {
         return parser->returnCodeFrom(ret);
     }
 
     request.set_time_zone(QTimeZone::systemTimeZoneId().toStdString());
 
-    return request_launch();
+    auto ret = request_launch();
+    if (ret == ReturnCode::Ok && request.instance_name() == petenv_name.toStdString())
+    {
+        const auto mount_source = QDir::toNativeSeparators(QDir::homePath());
+        const auto mount_target = QString{"%1:%2"}.arg(petenv_name, mp::home_automount_dir);
+        cout << fmt::format("Now auto-mounting '{}' in '{}'\n", mount_source, mount_target);
```

Comment:
I think this reads a tad better:

```suggestion
        cout << fmt::format("Mounting '{}' into '{}'\n", mount_source, mount_target);
```

Reply:
I was about to commit but I thought we will probably have a setting with "automount" in the name? If so, saying "auto-mounting" here might help the user make the connection. Please confirm if you don't think that's relevant so I can commit.

Reply:
I think "auto-" is implied here, since it happens on its own :)

Saying "auto-mounting" here suggests it differs from "mounting", so I'd stick to the latter.

Reply:
But see below!

Reply:
Ack, I will incorporate below.